docs/root/api-v1/tracing.rst

```diff
@@ -67,3 +67,23 @@ collector_endpoint
   *(optional, string)* The API endpoint of the Zipkin service where the
   spans will be sent. When using a standard Zipkin installation, the
   API endpoint is typically `/api/v1/spans`, which is the default value.
+
+Dynamic driver
+--------------
+
+.. code-block:: json
+
+  {
+    "type": "dynamic",
```

Comment:
Do we need v1 support? Unless there is a strong need for this at your end or by folks in the community, we're trying to feature freeze v1 and encourage new features to be v2 first.

Reply:
+1 if possible can we drop this.

Reply:
I removed the v1 support.